backend/main/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "typedoc": "typedoc",
     "typedoc:watch": "typedoc --watch",
-    "test": "vitest"
+    "test": "vitest",
+    "test:run": "vitest run"
```

Comment:
The default mode of the vitest is watch mode. The "run" commands prevent it and after finishing the test, stop the execution.

https://vitest.dev/guide/cli#vitest-run